### DISCUSSION TITLE
An actual Emergency shuttle

### DISCUSSION THREAD

**Discussion by Hoolny** [space-wizards/space-station-14#4108] [Feature Request]
When the game ends by emergency shuttle we just wait 4 minutes and it shows some text very anticlimactic and boring the emergency shuttle will be coming sooner or later but im wondering if any one is working on it or if someone was planning on adding it

**Comment by PaulRitter** [comment]
Yes its planned. While currently theoretically possible its blocked performancewise by some physics-optimisations.
